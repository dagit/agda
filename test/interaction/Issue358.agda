module Issue358 where

postulate Sigma : Set -> (Set -> Set) -> Set
syntax Sigma A (\x -> B)  = x colon A operator B

postulate T : Set
test = x colon T operator {!!}


