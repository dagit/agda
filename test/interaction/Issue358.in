top_command (cmd_load currentFile [])

-- Case split
goal_command 0 cmd_give "T -> T"
