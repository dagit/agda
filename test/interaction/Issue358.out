agda2_mode_code (agda2-status-action "")
agda2_mode_code (agda2-status-action "")
agda2_mode_code (agda2-info-action "*All Goals*" "?0 : Set\n" nil)
agda2_mode_code (agda2-goals-action '(0))
agda2_mode_code (agda2-give-action 0 "(T â†’ T)")
agda2_mode_code (agda2-status-action "")
agda2_mode_code (agda2-info-action "*All Goals*" "" nil)
agda2_mode_code (agda2-goals-action '())
